edb/server/pgcluster.py

```diff
@@ -419,7 +404,11 @@ def start(self, wait=60, *, server_settings=None, port, **opts):
                     self._data_dir))
 
         extra_args = ['--{}={}'.format(k, v) for k, v in opts.items()]
-        extra_args.append('--port={}'.format(port))
+
+        # We connect to our PostgreSQL server via a Unix socket,
+        # with TCP disabled. Which means that we don't really
+        # care about the port number being unique or even available.
+        extra_args.append('--port=65535')
```

Comment:
The Unix socket-only reasoning should allow us to run with the default port as well.  I don't see a reason why this is necessary.